adminer/include/adminer.inc.php

```diff
@@ -908,12 +908,15 @@ function dumpHeaders($identifier, $multi_table = false) {
 		return $ext;
 	}
 
-	/** Set the path of the file for webserver load
-	* @return string path of the sql dump file
-	*/
-	function importServerPath() {
-		return "adminer.sql";
-	}
+
+    /**
+     * List of files to import
+     * @return array
+     */
+    function importFiles() {
+        return glob(__DIR__ .'/*.{sql}', GLOB_BRACE);
```

Comment:
It looks like you don't actually need the braces here.

Also, last I checked, musl libc (used by Alpine, which is popular in the docker world) doesn't support `GLOB_BRACE`, so it would definitely be worth avoiding.

Reply:
Thanks, i changed the function